manifests/c/Cisco/UmbrellaRoamingClient/3.0.466/Cisco.UmbrellaRoamingClient.locale.en-US.yaml

```diff
@@ -0,0 +1,30 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Cisco.UmbrellaRoamingClient
+PackageVersion: 3.0.466
+PackageLocale: en-US
+Publisher: Cisco
+PublisherUrl: https://umbrella.cisco.com
+PublisherSupportUrl: https://umbrella.cisco.com/support
+PrivacyUrl: https://www.cisco.com/c/en/us/about/legal/privacy-full.html
+Author: Cisco
+PackageName: Umbrella Roaming Client
+PackageUrl: https://www.cisco.com/c/en/us/products/security/umbrella/umbrella-roaming.html
+License: Cisco Software License Agreement
+LicenseUrl: https://www.cisco.com/c/dam/en_us/about/doing_business/legal/docs/tc-us.pdf
+Copyright: Cisco 2024
+CopyrightUrl: https://www.cisco.com/c/en/us/about/brand-center/copyright-use.html

```

Comment:
> Man.... those are just Cisco's webpages. I'll go report false positives 😐

Hi @s-crypt,

Thanks for reporting those. To expedite this PR, it might be faster to remove these while the false positive report is processed. 

```suggestion
# PrivacyUrl:
Author: Cisco
PackageName: Umbrella Roaming Client
# PackageUrl: 
License: Cisco Software License Agreement
# LicenseUrl: 
Copyright: Cisco 2024
# CopyrightUrl: 
```